code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -40,6 +40,14 @@
 	mag_type = /obj/item/ammo_box/magazine/m50
 	can_suppress = 0
 
+/obj/item/weapon/gun/projectile/automatic/pistol/enforcer45
+	name = "Enforcer .45"
+	desc = "A pistol of modern design. Standard issue for Nanotrasen Asset Protection units."

```

Comment:
Something else as per lore, perhaps?

I don't think there is "Asset Protection" on Paradise.